CHANGES.md

```diff
@@ -1,6 +1,6 @@
 # 1.0.10
 
--adds NPM keywords to help people actually finds the package
+-adds NPM keywords to help people actually find the package
 -renames removeLocationStream to removeLocation. the former name was an artifact of an older version of this package and since it doesn't stream anything anymore, i figured to take it out to avoid confusion.
 -uncomment unused arrayBufferToBase64 helper function. it was commented out because we didn't need it but presumably some client might
```

Comment:
```suggestion
- Uncomments unused `arrayBufferToBase64` helper function. It was commented out because we didn't need it but presumably some client might.
```